assets/browser-fetch.js.t

```diff
@@ -1,7 +1,14 @@
 (function (originalGlobal) {
-  define('fetch', ['exports'], function(exports) {
+  if (!require.has('ember')) {
+    define('ember', ['exports'], function(exports) {
```

Comment:
Hmm, this seems not great. Defining the module could break other code / addons trying to sniff the module their own purposes. 🤔

Reply:
Since we are templating this file, we could tell it if we are on modules ember or not (either adding the `ember` module as a dep or falling back to global ember automatically).